courses/machine_learning/deepdive2/feature_engineering/solutions/Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2721 @@
+{
```

Comment:
<p>I'm not following the note to the lab reviewers here. I wouldn't be looking at this code if I didn't already clone, right? Or does you team use it in some other way?</p><p> </p><p>I'm also having trouble seeing the data here. Is there another notebook that generates this data? If yes, link to it here and tell students to run it so they know where to get the data. Otherwise, store the data in a cloud storage bucket and have students copy it into their environment using the gsutil or curl command.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Excellent point.  In the Coursera courses, there is a section in the  Qwik Labs setup portion where they clone the training-data-analyst repo before entering the AI Platform Notebook.  However, since ASL does not use Qwiklabs (I think), I'll keep things "in the notebook" -  I'll store the data in a storage bucket.  Is there a specific ASL bucket that your team already uses?</p>